website/docs/backends/types/remote.html.md

```diff
@@ -119,6 +119,27 @@ data "terraform_remote_state" "foo" {
 }
 ```
 
+## Example configuration using CLI input
+
+```hcl
+# main.tf 
+terraform {
+  required_version = "~> 0.12.0"
+
+  backend "remote" {}
+}
+
+# backend.file
```

Comment:
To avoid confusing an inattentive reader, I think we've usually preferred to show the contents of separate files as separate code blocks, with normal paragraph-level text between the two introducing what each file is for and (where it matters) what the file should be named. Otherwise it's easy for someone to misunderstand and think they need to create a single file containing the full contents of this code block, including the two comments.